src/main/java/org/jabref/preferences/ExporterFactoryFactory.java

```diff
@@ -0,0 +1,20 @@
+package org.jabref.preferences;
+
+import java.util.Map;
+
+import org.jabref.logic.exporter.ExporterFactory;
+import org.jabref.logic.exporter.SavePreferences;
+import org.jabref.logic.exporter.TemplateExporter;
+import org.jabref.logic.journals.JournalAbbreviationLoader;
+import org.jabref.logic.layout.LayoutFormatterPreferences;
+
+public class ExporterFactoryFactory {
+
+    public static ExporterFactory create(JabRefPreferences preferences, JournalAbbreviationLoader abbreviationLoader) {
```

Comment:
What about moving this method to `JabRefPreferences` as an instance method `createExportFactory(abbrLoader)`?